packages/shared/src/BootcAPI.ts

```diff
@@ -17,14 +17,15 @@
  l***********************************************************************/
 
 import type { BootcBuildInfo, BuildType } from './models/bootc';
-import type { ImageInfo, ImageInspectInfo } from '@podman-desktop/api';
+import type { ImageInfo, ImageInspectInfo, ManifestInspectInfo } from '@podman-desktop/api';
 
 export abstract class BootcApi {
   abstract checkPrereqs(): Promise<string | undefined>;
   abstract buildExists(folder: string, types: BuildType[]): Promise<boolean>;
   abstract buildImage(build: BootcBuildInfo, overwrite?: boolean): Promise<void>;
   abstract pullImage(image: string): Promise<void>;
   abstract inspectImage(image: ImageInfo): Promise<ImageInspectInfo>;
+  abstract inspectManifest(image: ImageInfo): Promise<ManifestInspectInfo>;
```

Comment:
Discussed with Charlie - I don't like this (vs having engineId, id parameters), but it is convenient for UI and consistent with other functions here. API is not final, so fine for now at least.